docroot/modules/custom/mass_alerts/mass_alerts.routing.yml

```diff
@@ -22,3 +22,19 @@ mass_alerts.user_autocomplete:
     _format: json
   requirements:
     _permission: 'access email address list for alerts config page'
+
+
+mass_alerts.site_alerts:
+  path: '/jsonapi/site-alerts'
```

Comment:
I'll be curious to see what others think about using /jsonapi as a base route. I would have probably just set a route like /ma-alerts/site-alerts or something similar, but I think this is okay.

Reply:
I also dont like this name. We are not using jsonapi nor returning JSON. I would suggest /alerts/sitewide and /alerts/page/[n]